config/routes.rb

```diff
@@ -116,8 +116,9 @@
 
   # countries
   get '/countries' => 'countries#index'
-  get '/country/*all' => 'country#index', as: :country
   get '/countries/overview' => 'countries#overview'
+  get '/country/embed/:widget/:iso(/:region)(/:sub_region)' => 'country#embed'
```

Comment:
Why can't we use the react router for this? Then we only need one entry point?

Reply:
because we have the header & footer loaded on rails, and the embed landing doesn't need it. So we need to load a specific view in rails.

Reply:
Ah of course. Well done.